src/components/v5/common/TableWithBurgerMenu/types.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  FieldValues,
+  UseFieldArrayAppend,
+  UseFieldArrayRemove,
+  UseFormGetValues,
+} from 'react-hook-form';
+import { TableProps } from '../Table/types';
+
+export interface TableWithBurgerMenuProps
+  extends Pick<TableProps, 'tableTitle' | 'columns'> {
+  actions: {
+    fields: Record<'id', string>[];
```

Comment:
I don't understand why is this prop nested here. Why aren't you using the interface from the base table?